### PR TITLE
chore: remove old styling

### PR DESCRIPTION
**Pull request by droshev**
remove old styling

### CODE REVIEW

File: libs/core/src/lib/avatar-group/avatar-group.component.scss [hunk 1]
```diff
@@ -1,769 +1 @@
...
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-10 {
-    background-color: #647987;
-    background-color: var(--sapAccentColor10,#647987);
-    border-color: #647987;
-    border-color: var(--sapAccentColor10,#647987);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
+@import '~fundamental-styles/dist/avatar-group';
```
Comment (JKMarkowski) [inline libs/core/src/lib/avatar-group/avatar-group.component.scss:1]: `0.16.1` version doesn't bring valid styles. Let's for `0.17.0` styles migration
image

File: libs/core/src/lib/table/table.component.scss [hunk 2]
```diff
@@ -14,11 +14,6 @@
 	}
 }
 
-// Remove after merging and migrate https://github.com/SAP/fundamental-styles/pull/2192
```
Comment (JKMarkowski) [inline libs/core/src/lib/table/table.component.scss:17]: Table changes will come with `0.17.0`, so let's uncommit this file as well

Comment (droshev) [reply libs/core/src/lib/table/table.component.scss:17]: we brought the latest https://github.com/SAP/fundamental-ngx/blob/7940614e37ba0f49eb746ab2fcb887c98fe83db0/package.json#L116

Comment (droshev) [reply libs/core/src/lib/table/table.component.scss:17]: i reverted it

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**